app/views/planning_applications/recommendation_form.html.erb

```diff
@@ -30,26 +31,19 @@
             <%= form.radio_button :decision, "refused", class: "govuk-radios__input", id: "status-refused-conditional", "aria-controls": "conditional-status-refused-conditional", "aria-expanded": "false" %>
             <%= form.label :decision_refused, "No", class: "govuk-label govuk-radios__label", for: "status-refused-conditional" %>
           </div>
-          <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-status-refused-conditional">
-            <div class="govuk-form-group">
-              <p class="govuk-body">
-                <br>
-                <strong>Please provide which GDPO policy (or policies) have not been met. </strong>
-                If there are multiple, please separate by a comma.
-              </p>
-              <p class="govuk-body">
-                <span class="govuk-hint">
-                  For example, "GPDO 2015 S.2 P.1 A.1 (f)(i), GPDO 2015 S.2 P.1 A.1 (f)(ii)"
-                </span>
-              </p>
-              <p class="govuk-body">
-                This <strong>will</strong> appear on the decision notice.
-              </p>
-              <%= form.text_area :public_comment, class: "govuk-textarea", id: "public_comment", rows: "3", "aria-describedby": "public-comment-hint" %>
-            </div>
-          </div>
         </div>
       </div>
+     </div>
+      <div class="govuk-form-group">
+        <p class="govuk-body">
+          <br>
+          <strong>Please provide supporting information to indicate which GDPO policy(s) have or have not been met. </strong>
+        </p>
+        <p class="govuk-body">
+          This information <strong>will</strong> appear on the decision notice.
+        </p>
+        <%= form.text_area :public_comment, class: "govuk-textarea govuk-!-margin-bottom-9", id: "public_comment", rows: "3", "aria-describedby": ":public-comment-hint" %>
```

Comment:
We no longer provide a hint text, so I don't think we should have `"aria-describedby": ":public-comment-hint"` on this any more?

Reply:
In fact, this text_area doesn't have a label for it, which doesn't seem right. I think it's been copied and pasted a few too many times and lost some of its semantics. Could you fix it up so it matches https://design-system.service.gov.uk/components/textarea/?

Reply:
I took the hint out because that's how the design is asking for it to be now:

![image](https://user-images.githubusercontent.com/9452321/108737568-a0b19180-752a-11eb-87d6-2dca9fff3b00.png)

Reply:
on the aria-described by, I had understood it was used for accessibility purposes to help screen readers relate parts of content to the other - and because we still have aria-describedby in the "private comment" ?